docs/previews/README.md

```diff
@@ -110,120 +114,120 @@ export default function App({
+`initializeNextServerSideProps` is used to allow for Server Side Rendering. It knows how to get URL information on the server so that we can query WP and pull the right `pageInfo` on the initial request. This is critical for SEO. We want to return the rendered page on the first request so that search engines can index our content.
 
 `/lib/components/Post.tsx`
 
 ```jsx
-import React from "react";
-import { usePost } from "@wpengine/headless";
+import React from 'react';
+import { usePost } from '@wpengine/headless';
 
 export default function Post() {
-	const post = usePost();
-
-	return (
-		<div>
-			{post && (
-				<div>
-					<div>
-						<h5>{post.title}</h5>
-						<p
-							dangerouslySetInnerHTML={{
-								__html: post.content ?? "",
-							}}
-						/>
-					</div>
-				</div>
-			)}
-		</div>
-	);
+  const post = usePost();
+
+  return (
+    <div>
+      {post && (
+        <div>
+          <div>
+            <h5>{post.title}</h5>
+            <p
+              dangerouslySetInnerHTML={{
+                __html: post.content ?? '',
+              }}
+            />
+          </div>
+        </div>
+      )}
+    </div>
+  );
 }
 ```
 
 `/lib/ components/Posts.tsx`
 
 ```jsx
-import React from "react";
-import Link from "next/link";
-import { usePosts } from "@wpengine/headless";
+import React from 'react';
+import Link from 'next/link';
+import { usePosts } from '@wpengine/headless';
 
 export default function Posts() {
-	const posts = usePosts();
-
-	return (
-		<div>
-			{posts &&
-				posts.map((post) => (
-					<div key={post.id} id={`post-${post.id}`}>
-						<div>
-							<Link href={post.uri}>
-								<h5>
-									<a href={post.uri}>{post.title}</a>
-								</h5>
-							</Link>
-							<p
-								dangerouslySetInnerHTML={{
-									__html: post.excerpt ?? "",
-								}}
-							/>
-						</div>
-					</div>
-				))}
-		</div>
-	);
+  const posts = usePosts();
+
+  return (
+    <div>
+      {posts &&
+        posts.map((post) => (
+          <div key={post.id} id={`post-${post.id}`}>
+            <div>
+              <Link href={post.uri}>
+                <h5>
+                  <a href={post.uri}>{post.title}</a>
+                </h5>
+              </Link>
+              <p
+                dangerouslySetInnerHTML={{
+                  __html: post.excerpt ?? '',
+                }}
+              />
+            </div>
+          </div>
+        ))}
+    </div>
+  );
 }
 ```
 
 #### Configuration
 
 We need to let the frontend know about our WordPress instance. The framework expects a few environment variables with this information. Create a file in the root of the project `/.env.local`.
 
-```
+```bash
 # Base URL for WordPress
-NEXT_PUBLIC_WORDPRESS_URL=http://developer-hub.local
+NEXT_PUBLIC_WORDPRESS_URL=http://yourwp.com
```

Comment:
http://yourwp.com resolves to the Washington Post.

Maybe use example.com here instead?

```suggestion
NEXT_PUBLIC_WORDPRESS_URL=https://example.com
```